entries/t/tu.berlin.json

```
@@ -0,0 +1,19 @@
+{
+  "Technische Universität Berlin": {
+    "domain": "tu.berlin",
+    "additional-domains": [
+      "tu-berlin.de"
+    ],
+    "tfa": [
+      "totp",
+      "sms"
+    ],
+    "documentation": "https://www.campusmanagement.tu-berlin.de/menue/dienste/konto_karte/tan_verfahren/",
```

Maybe we should note this exception:
```suggestion
    "documentation": "https://www.campusmanagement.tu-berlin.de/menue/dienste/konto_karte/tan_verfahren/",
    "notes": "2FA is only available for selected services",
```

---

Looks good to me.